docs/wiki/Deploying-ALZ-ZTNetwork.md

```diff
@@ -0,0 +1,49 @@
+## Azure landing zone portal accelerator deployment with Zero Trust network principles
+
+This section describes how to deploy the Azure landing zone portal accelerator with a jump start on Zero Trust Networking Principles for Azure landing zones. For more information on Zero Trust security model and principles visit [Embrace proactive security with Zero Trust](/security/business/zero-trust). Let's review the [Zero Trust aligned networking](/security/zero-trust/deploy/networks) configurations in the [Azure landing zone portal accelerator](/azure/cloud-adoption-framework/ready/landing-zone/#azure-landing-zone-portal-accelerator).
+
+## Deploy networking topology and Connectivity
+
+On the "Network Topology and Connectivity" section of the Azure landing zone portal accelerator select "Hub and spoke with Azure Firewall" radio button. Next, Select the platform connectivity subscription from the drop down. Confirm or update the address space and first networking hub region, in this case East US.
+
+![image](https://user-images.githubusercontent.com/8091766/228360733-9713f5ff-dd53-4995-b309-220442f978b5.png)
+
+Hub and spoke is the best option for Zero Trust Organizations should not have one single connection in and out of their cloud network. Network should be segmented into smaller islands with their own ingress and egress controls in minimize "blast radius". 
```

Comment:
Is this going to get us in trouble with vWAN teams? 

Also the sentence does not make sense, please re-word. "Hub and spoke is the best option for Zero Trust **Organizations should** not have one single"

Reply:
@jtracey93 - There is some stuff going on with vWAN on the Zero Trust side.  I'll do a PR to adjust some of this to try and bring it in line.

Reply:
Try https://github.com/Azure/Enterprise-Scale/pull/1267/files to see if you agree with these changes.